src/components/molecules/VisualisationPreview.tsx

```diff
@@ -99,28 +88,17 @@ const VisualisationPreview: React.FC<VisualisationPreviewProps> = ({ metadata, s
                     <Heading size={400}>
                         Forhåndsvisning
                         <Tooltip content="Forhåndsvisningen er kun ment som referanse. Størrelse og endelig data vil endres i dashbordet.">
-                            <InfoSignIcon color="disabled" marginLeft={12} marginTop={3} />
+                            <InfoSignIcon color="disabled" marginLeft={12} marginTop={3} verticalAlign="sub" />
                         </Tooltip>
                     </Heading>
                     <Pane flex="1" />
-                    <AddToDashboard
-                        dashboardItemInfo={{
-                            size,
-                            name: metadata.name,
-                            datasourceId: metadata.datasourceId,
-                            paragraph,
-                            id: metadata.id,
-                            query: WEATHER_MET_API,
-                            visualisationType: selectedVisualisation,
-                        }}
-                    />
                 </Pane>
-                <Pane flex="1" height="100%">
+                <Pane backgroundColor="white">
```

Comment:
This leaves a white background on size medium or small. I think the white background should be set in dashboarditem instead of the container.

Reply:
I had some issues with that, but I'll take a look when I have time.